src/kirby/components/card/card-header/card-header.component.tns.scss

```diff
@@ -1,7 +1,7 @@
 @import '../../../scss/native-imports';
 
 .header {
-  margin: 16;
+  margin: font-size("s");
```

Comment:
Should we not use `size('s')` here instead?  size `s` is actual `16` where font-size `s` is `14`.